Source/Core/VideoBackends/OGL/TextureCache.h

```diff
@@ -8,54 +8,45 @@
 
 #include "Common/CommonTypes.h"
 #include "Common/GL/GLUtil.h"
+#include "VideoBackends/OGL/ProgramShaderCache.h"
 
 #include "VideoCommon/TextureCacheBase.h"
 #include "VideoCommon/VideoCommon.h"
 
+class AbstractTexture;
+struct TextureConfig;
+
 namespace OGL
 {
+
+// TODO: Don't do this
```

Comment:
A temporary fix would be to move them to OGLTexture.cpp as well, with an additional Init() and Shutdown() function.